packages/core/src/shared/constants.ts

```diff
@@ -138,3 +138,6 @@ export const amazonQHelpUrl = 'https://aws.amazon.com/q/'
 // URL for Amazon Q VS Code
 export const amazonQVscodeMarketplace =
     'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode'
+
+export const amazonQDiffScheme = 'amazon-q-view-diff'
```

Comment:
we probably need a single `schemes.fs` file somewhere, but until we decide that, this can at least live next to the existing ones: https://github.com/aws/aws-toolkit-vscode/blob/80ac38308e114797f8656d793056eb576dc77f12/packages/core/src/shared/constants.ts#L111-L114